src/deploy-airkeeper.ts

```
@@ -0,0 +1,65 @@
+import { join } from 'path';
+import { PromptObject } from 'prompts';
+import { OperationsRepository } from './types';
+import { runAndHandleErrors, runShellCommand } from './utils/cli';
+import { promptQuestions } from './utils/prompts';
+import { readOperationsRepository } from './utils/read-operations';
+
+const questions = (operationsRepository: OperationsRepository): PromptObject[] => {
+  return [
+    {
+      type: 'autocomplete',
+      name: 'name',
+      message: 'Which API Integration do you want to deploy?',
+      choices: Object.keys(operationsRepository.apis).map((api) => ({ title: api, value: api })),
+    },
+    {
+      type: 'autocomplete',
+      name: 'deployment',
+      message: (prev) => `Which deployment of ${prev} do you want to deploy?`,
+      choices: (prev) =>
+        Object.keys(operationsRepository.apis[prev].deployments).map((deployment) => ({
+          title: deployment,
+          value: deployment,
+        })),
+    },
+  ];
+};
+
+const main = async () => {
+  const airkeeperVersion = require('@api3/airkeeper/package.json').version;
+  const operationsRepository = readOperationsRepository();
+  const response = await promptQuestions(questions(operationsRepository));
+
+  const deploymentDirectory = join(__dirname, '..', 'data', 'apis', response.name, 'deployments', response.deployment);
+  const awsSecretsFilePath = join(deploymentDirectory, 'aws.env');
+
+  const config = operationsRepository.apis[response.name].deployments[response.deployment].config;
+  const stage = config.nodeSettings.stage;
+  const region =
+    config.nodeSettings.cloudProvider.type === 'aws' ? config.nodeSettings.cloudProvider.region : 'us-east-1';
```

This may be a mistake.
`config.nodeSettings.cloudProvider.type` here can be 'aws' or 'gcp'.
GCP regions are like `us-east1` and AWS regions are like `us-east-1`.

But I think instead of assuming the region you should throw an error if the region is unset.